astroid/raw_building.py

```diff
@@ -391,9 +391,6 @@ def _astroid_bootstrapping(astroid_builtin=None):
 
 _astroid_bootstrapping()
 
-# TODO : find a nicer way to handle this situation;
```

Comment:
Why was this TODO removed, is this the same bug?

Reply:
I thought so because of the semicolon referencing the recursion issue on the following line